cogeo_mosaic/backends/dynamodb.py

```diff
@@ -136,21 +172,20 @@ def update(
 
         self._update_metadata()
 
-        return
-
-    def _create_table(
-        self, overwrite: bool = False, billing_mode: str = "PAY_PER_REQUEST"
-    ):
-        if self._table_exist():
-            if not overwrite:
-                raise MosaicExists("Table already exist, use `overwrite=True`.")
-            self.table.delete()
-            self.table.wait_until_not_exists()
+    def _create_table(self, billing_mode: str = "PAY_PER_REQUEST", **kwargs: Any):
+        """Create DynamoDB Table."""
+        logger.debug(f"Creating {self.table_name} Table.")
 
         # Define schema for primary key
         # Non-keys don't need a schema
-        attr_defs = [{"AttributeName": "quadkey", "AttributeType": "S"}]
-        key_schema = [{"AttributeName": "quadkey", "KeyType": "HASH"}]
+        attr_defs = [
+            {"AttributeName": "mosaicId", "AttributeType": "S"},
+            {"AttributeName": "quadkey", "AttributeType": "S"},
+        ]
+        key_schema = [
+            {"AttributeName": "mosaicId", "KeyType": "RANGE"},
+            {"AttributeName": "quadkey", "KeyType": "HASH"},
+        ]
```

Comment:
@leothomas I'm getting 
> ClientError: An error occurred (ValidationException) when calling the CreateTable operation: Invalid KeySchema: The first KeySchemaElement is not a HASH key type

any idea?

Reply:
Could it be just switching the order of the two elements in the list? (ie: 
```
key_schema = [
            {"AttributeName": "quadkey", "KeyType": "HASH"},
            {"AttributeName": "mosaicId", "KeyType": "RANGE"},
        ]
```
)

Reply:
yep tried and it worked at least for creating the table. Now I have another error 🎉

Reply:
Note that in the change, Vincent switched the index types, not the order of the keys.